### PR TITLE
Kick off file conversion after GRIN download

### PR DESCRIPTION
**Pull request by priya-pai**
## Describe your changes
This PR kicks off the file conversion step function after a book is downloaded and extracted from GRIN. It also adds the file conversion kick off to the NYPL API Manager to keep the API code together.

This PR also takes care of a small GRIN conversion edge case where books that were updated as converted but not yet in our DB would be passed over.

## How to test
`python3 -m processes.grin.download --barcode=33433115534533` (This is a new barcode extracted and uploaded to our QA bucket)

### CODE REVIEW

File: etl-pipeline/processes/grin/download.py [hunk 1]
```diff
@@ -23,12 +23,25 @@ def __init__(self, barcode):
...
 
+        self.nypl_api_manager = NYPLAPIManager()
+
     def run_process(self):
         with DBManager() as self.db_manager:
             file_content = self.download_and_upload_book()
 
             self.unpack_and_upload_ocr_files(file_content)
 
+            mets_key = f"grin/{self.barcode}/NYPL_{self.barcode}.xml"
```
Comment (kylevillegas93) [inline etl-pipeline/processes/grin/download.py:34]: yay!

**Review by kylevillegas93 — APPROVED** [review]
alright!! integrated quite nicely
